src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -117,64 +117,64 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="ColorScheme_Background.Text" xml:space="preserve">
+  <data name="ColorScheme_Background.[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip" xml:space="preserve">

```

Comment:
why is this one (and others) `[using:Windows.UI.Xaml.Controls]ToolTipService.ToolTip` while the rest are just `ToolTip`?

Reply:
Because that one is linked to the "Background" button (`x:Uid="identifier"`), while the rest are linked through the code.